tests/data/README.md

```diff
@@ -6,6 +6,8 @@
 
 ![](circle-trns-after-plte.png)
 
+License: [same as libavif](https://github.com/AOMediaCodec/libavif/blob/main/LICENSE)
```

Comment:
Perhaps we can add a notice at the beginning of the README file to make this license the default.